From 2d3799832f0785558bc54a81229e254030c82e83 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Mon, 5 Sep 2011 14:05:13 +0800
Subject: [PATCH] powerpc/p2020: add several i2c device nodes

- Add Analog ADT7461 on p2020rdb.
- Add ZL2006 on p2020DS.
- Add several devices which is already in 32bit DTS, but missed
  on 36bit DTS.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/boot/dts/p2020ds.dts      |    5 +++++
 arch/powerpc/boot/dts/p2020ds_36b.dts  |   20 ++++++++++++++++++++
 arch/powerpc/boot/dts/p2020rdb.dts     |    5 +++++
 arch/powerpc/boot/dts/p2020rdb_36b.dts |   16 ++++++++++++++++
 4 files changed, 46 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/p2020ds.dts b/arch/powerpc/boot/dts/p2020ds.dts
index b715442..2181973 100644
--- a/arch/powerpc/boot/dts/p2020ds.dts
+++ b/arch/powerpc/boot/dts/p2020ds.dts
@@ -256,6 +256,11 @@
 			interrupt-parent = <&mpic>;
 			dfsrr;
 
+			zl6100@11{
+				compatible = "intersil,zl6100";
+				reg = <0x11>;
+			};
+
 			rtc@68 {
 				compatible = "dallas,ds1339";
 				reg = <0x68>;
diff --git a/arch/powerpc/boot/dts/p2020ds_36b.dts b/arch/powerpc/boot/dts/p2020ds_36b.dts
index 5df895e..3115754 100644
--- a/arch/powerpc/boot/dts/p2020ds_36b.dts
+++ b/arch/powerpc/boot/dts/p2020ds_36b.dts
@@ -214,6 +214,16 @@
 			interrupts = <43 2>;
 			interrupt-parent = <&mpic>;
 			dfsrr;
+
+			eeprom@50 {
+				compatible = "atmel,24c64";
+				reg = <0x50>;
+			};
+
+			eeprom@51 {
+				compatible = "atmel,24c64";
+				reg = <0x51>;
+			};
 		};
 
 		i2c@3100 {
@@ -225,6 +235,16 @@
 			interrupts = <43 2>;
 			interrupt-parent = <&mpic>;
 			dfsrr;
+
+			zl6100@11{
+				compatible = "intersil,zl6100";
+				reg = <0x11>;
+			};
+
+			rtc@68 {
+				compatible = "dallas,ds1339";
+				reg = <0x68>;
+			};
 		};
 
 		serial0: serial@4500 {
diff --git a/arch/powerpc/boot/dts/p2020rdb.dts b/arch/powerpc/boot/dts/p2020rdb.dts
index 4887f51..58ab00d 100644
--- a/arch/powerpc/boot/dts/p2020rdb.dts
+++ b/arch/powerpc/boot/dts/p2020rdb.dts
@@ -202,6 +202,11 @@
 			interrupt-parent = <&mpic>;
 			dfsrr;
 
+			adt7461@4c{
+				compatible = "adi,adt7461";
+				reg = <0x4c>;
+			};
+
 			eeprom@50 {
 				compatible = "atmel,24c64";
 				reg = <0x50>;
diff --git a/arch/powerpc/boot/dts/p2020rdb_36b.dts b/arch/powerpc/boot/dts/p2020rdb_36b.dts
index e614c09..acee00e 100644
--- a/arch/powerpc/boot/dts/p2020rdb_36b.dts
+++ b/arch/powerpc/boot/dts/p2020rdb_36b.dts
@@ -201,6 +201,17 @@
 			interrupts = <43 2>;
 			interrupt-parent = <&mpic>;
 			dfsrr;
+
+			adt7461@4c{
+				compatible = "adi,adt7461";
+				reg = <0x4c>;
+			};
+
+			eeprom@50 {
+				compatible = "atmel,24c64";
+				reg = <0x50>;
+			};
+
 			rtc@68 {
 				compatible = "dallas,ds1339";
 				reg = <0x68>;
@@ -216,6 +227,11 @@
 			interrupts = <43 2>;
 			interrupt-parent = <&mpic>;
 			dfsrr;
+
+			eeprom@52 {
+				compatible = "atmel,24c64";
+				reg = <0x52>;
+			 };
 		};
 
 		serial0: serial@4500 {
-- 
1.7.0.4

